.gitignore

```diff
@@ -10,3 +13,38 @@
 /bin
 /etc/ssl/selfsigned-*
 */miniconda3
+
+
+# vault
+lib/authz/vault/data/*
+lib/authz/vault/logs/*
+lib/authz/vault/policies/*
+lib/authz/vault/config/*
+lib/authz/vault/keys.txt
+
+# keycloak
+lib/authz/keycloak/data/*
+lib/authz/keycloak/data/secrets.env
+
+# tyk
+lib/authz/tyk/data/*
+
+# opa 
+# keep global policies
+#lib/opa/*
+lib/authz/opa/*
+lib/authz/opa/policy.rego
+
+lib/candig_server/authz/opa/*.rego
```

Comment:
not needed since you have opa/*

Reply:
not necessarily
`lib/candig_server/authz` and `lib/authz` are different directories